### ISSUE TITLE
Filtering specific system call arguments

### ISSUE THREAD

**Issue by Manouchehri** [cloudflare/sandbox#15]
As mentioned in the `README.md`, currently this tool doesn't support filtering arguments.

Any progress made on adding support for that yet? =)

**Comment by ignatk** [comment]
Not really, as this feature would probably be rather complicated to implement in a generic way as we would need to have some kind of a domain specific config language to express the constraints on the system call arguments.

Moreover, at least from the top of my head, I don't see it flexible enough for most system calls, except the ones which just have a file path as one of its arguments. Many system calls accept some mostly ephemeral process parameters, like file descriptors or pointers, so not sure how can one easily express those in the config language.
